PublicGitArchive/pga/cmd/root.go

```diff
@@ -31,7 +31,7 @@ For more info, check http://pga.sourced.tech/`,
 		}
 
 		pgaVersion = pv
-		indexName = pv + ".csv.gz"
+		indexName = pv + ".index.csv.gz"
```

Comment:
Is this compatible with the siva dataset?

Reply:
[Not yet](https://github.com/src-d/infrastructure/issues/1170#issuecomment-539444851) but it seemed better